src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -409,16 +411,16 @@ private static boolean shouldBeAloneOnLineWithNotAloneOption(Details details,
      * Checks whether right curly is alone on a line.
      *
      * @param details for validation.
-     * @param targetSrcLine A string with contents of rcurly's line
+     * @param targetSrcLine Unicode code points array with contents of rcurly's line
```

Comment:
Please do `Unicode` -> `unicode`, javadoc tag description should not be capitalized if it is not a complete sentence. There are a few instances in this PR.

Reply:
done

Reply:
Resolved.